.circleci/config.yml

```
@@ -190,8 +195,14 @@ jobs:
       - run:
           name: Deploy prisma
           working_directory: server/scripts/prisma_deploy_all/
-          command: node index.js
-
+          command: |
+            if ${{ pipeline.parameters.prismaDeployForce }}; then
+              echo "Deploying prisma with --force"
+              node index.js --force
+            else
+              echo "Deploying prisma without --force"
+              node index.js
+            fi
```

Il y a une syntaxe pour les paramètres CircleCI qui permet de simplifier ça
```suggestion
          command: node index.js <<#pipeline.parameters.prismaDeployForce>>--force<</pipeline.parameters.prismaDeployForce>>
```